database/migrations/2021_03_27_160120_add_indexes_to_network_aircraft.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIndexesToNetworkAircraft extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('network_aircraft', function (Blueprint $table) {
+            $table->index('altitude');
```

Comment:
Is this an index on just those columns, e.g. you're querying on just altitude, or would be be useful to have them as compound indexes?

Reply:
Compounded it :)